src/NServiceBus.AcceptanceTests/BestPractices/When_subscribing_to_command_bestpractices_disabled_on_endpoint.cs

```diff
@@ -42,9 +42,8 @@ public Endpoint()
                 {
                     var routing = c.UseTransport(r.GetTransportType()).Routing();
                     routing.DoNotEnforceBestPractices();
-                })
-                    .AddMapping<MyCommand>(typeof(Endpoint))
-                    .AddMapping<MyEvent>(typeof(Endpoint));
+                }, metadata => metadata.RegisterPublisherFor<MyCommand>(typeof(Endpoint)))
+                .AddMapping<MyEvent>(typeof(Endpoint));
```

Comment:
the test which requires this mapping doesn't seem like it should be here. Isn't this the same test as https://github.com/Particular/NServiceBus/pull/4246/files#diff-8451bba86d6dfa36a0e4daa577b2eeadR11 ?